tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -11,7 +11,7 @@
 import uuid
 from contextlib import nullcontext as does_not_raise
 from functools import partial
-from typing import Any, Callable, Optional, Sequence, Union
+from typing import Any, Callable, Dict, Optional, Sequence, Union
```

Comment:
We don't need to import Dict anymore with 3.9

Reply:
removed